index.html

```diff
@@ -34,5 +34,131 @@
       </li>
     </ul>
     </div>
+    <div id="recent">
```

Comment:
Great work @Solobarine 👏

- [ ] [Optional] You may try to fix this white space too. It is not how it is supposed to look according to the Figma design

![spacing](https://user-images.githubusercontent.com/63917725/177011139-4e448e07-d06b-49a0-893d-be34391b0f57.PNG)